lib/tasks/create_test_accounts.rb

```diff
@@ -15,12 +15,10 @@ def create_account(email: 'joe.smith@email.com', password: 'salty pickles', mfa_
   user = User.create!(email: email)
   user.skip_confirmation!
   user.reset_password(password, password)
-  user.phone = mfa_phone || phone
-  user.phone_confirmed_at = Time.zone.now
   user.save!
   user.create_phone_configuration(
     phone: mfa_phone || phone,
-    confirmed_at: user.phone_confirmed_at,
+    confirmed_at: user.Time.zone.now,
```

Comment:
Did you mean `Time.zone.now` without `user.` before it?

Reply:
Doh. Good catch.